modules/internal/Atomics.chpl

```diff
@@ -187,6 +187,21 @@ module Atomics {
       atomic_init(_v, default);
     }
 
+    pragma "no doc"
+    proc init=(other : AtomicBool) {
+      this._v = other._v;
+    }
+
+    pragma "no doc"
+    proc init=(other : bool) {
```

Comment:
I didn't see much PR message but this new atomics support would be worth calling out in the PR message